README.md

```diff
@@ -1,7 +1,29 @@
 # nf-model2data
 This is the nextflow workflow that runs model to data challenges.
 
+## Prerequisites
+
+In order to use this workflow, you must already have completed the following steps:
+
+1. Created a Synapse project shared with challenge participants.
+2. Created an evaluation queue within the Synapse project.
+3. One or more Docker containers must have already been submitted to your evaluation queue.
```

Comment:
Hmm, what happens right now when there are no submissions?

Reply:
The same thing happens if you run it and there are submissions, but none where `status = 'RECEIVED'`